assets/dev/js/editor/regions/panel/commands/internal/state-ready.js

```diff
@@ -3,6 +3,13 @@ import CommandInternalBase from 'elementor-api/modules/command-internal-base';
 export class StateReady extends CommandInternalBase {
```

Comment:
Where this command triggered?

Reply:
After the editor full loaded, please check in code.`internal( 'panel/state-ready' )`